lib/core.js

```diff
@@ -6,12 +6,7 @@ function transformer(from, to, coords) {
   var transformedArray, out, keys;
   if (Array.isArray(coords)) {
     transformedArray = transform(from, to, coords);
-    if (coords.length === 3) {
-      return [transformedArray.x, transformedArray.y, transformedArray.z];
-    }
-    else {
-      return [transformedArray.x, transformedArray.y];
-    }
+    return [transformedArray.x, transformedArray.y].concat(coords.splice(2));
```

Comment:
why don't you put this where `return [transformedArray.x, transformedArray.y, transformedArray.z];` with the conditional being `coords.length > 2` that way the most common case isn't slowed down